proc/foxsi_level0_to_level1.pro

```diff
@@ -308,7 +371,7 @@ FUNCTION	FOXSI_LEVEL0_TO_LEVEL1, FILENAME, DETECTOR = DETECTOR, STOP = STOP, $
 					  data_struct.hit_cm[1] gt data_struct.hit_adc[1])
 		data_struct[check].error_flag = data_struct[check].error_flag + 16
 	
-		; different geometry for CdTe detectors.
+		; different geometry for CdTe detectors. Now that we changed the way the detector coordinates are calculated for CdTe this should be apply also to CDTE detectors
 		if not keyword_set( CDTE ) then begin
```

Comment:
From my understanding, this set of lines will apply both when CdTe is set to 0 (default) and when CdTe is set to 1. Should the "if" logic around this statement be removed to avoid confusion?

Reply:
I totally forgot about this. I made this comment to remind myself to look more closely at this. I think for now I do not want to remove the if logic because I am not actually sure that it should work for CdTe. I think I will create an issue for this specific problem, since I have to look at the definition of the error flags for CdTe anyway, and modify this line in a later iteration of the code, if that's ok.

Reply:
Sounds good!